business_requirement_deliverable_cost/models/business.py

```diff
@@ -46,10 +46,10 @@ def _compute_sale_price_total(self):
     @api.multi
     def _get_partner(self):
         self.ensure_one()
-        br_id = False
-        if self.business_requirement_deliverable_id:
+        br_id, br_deliverable = 2 * [False]
+        if self.business_requirement_deliverable_id.id:
             br_deliverable = self.business_requirement_deliverable_id
-        if br_deliverable and br_deliverable.business_requirement_id:
+        if br_deliverable and br_deliverable.business_requirement_id.id:
```

Comment:
with the new API, you can use:

```
if br_deliverable.business_requirement_id:
```
because the browse records implement the Null Object pattern.